worldedit-core/src/main/java/com/boydti/fawe/beta/implementation/blocks/CharBlocks.java

```diff
@@ -16,6 +16,13 @@
     protected static final Section FULL = new Section() {
         @Override
         public final char[] get(CharBlocks blocks, int layer) {
+            if (blocks.blocks[layer] == null) {
```

Comment:
Does that still return the expected values? Can a non-empty section be trimmed? Because if that's the case, processing chunks will end up in wrong calculations.